src/framework/module/nodeLayout/grid/excute.js

```
@@ -1,392 +1,277 @@
+  return typeof e === "string";
 }
-
-function isNumber(value) {
-    return typeof value === 'number';
+function isNumber(e) {
+  return typeof e === "number";
 }
-
-function isNaN (input) {
-    return Number.isNaN(Number(input))
+function isNaN(e) {
```

Biome (1.9.4)

[error] Do not shadow the global "isNaN" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)